src/context/reducers/analyseReducer.tsx

```diff
@@ -23,14 +10,12 @@ export type ISeries = {
   covidnetPluginId: number, 
   imageName: string,
   imageId: string,
-  predCovid: number,
-  predPneumonia: number,
-  predNormal: number,
+  columnNames: string[], // Holding classification classes/values in array
+  columnValues: number[],
```

Comment:
For my personal understanding, I was wondering what are advantages and disadvantages of keeping the prediction results in `columnNames` and `columnValues` arrays compared to keeping them in a dictionary where the key is the name of the column and the value is the number such as { predResults: {[key: string]: number}}